### DISCUSSION TITLE
Withdrawing xp from ender chests

### DISCUSSION THREAD

**Discussion by CodingPanda0623** [Stardust-Labs-MC/StardustMC#3] [Ideas]
Would it be possible to make withdrawing xp from ender chests a lot faster just like how storing xp in the echest can be sped up tremendously by crouching on top of it? Stood under my echest (with around 150k xp stored) around 15 minutes and only took out about half lol.

**Comment by GinnyEvilShadow** [comment]
In addition, maybe allow for the xp stored to mend tools and whatnot to make the stored xp more useful.

Although I understand if this is declined, just figured I'd put it out there.

**Reply by aisukuma** [reply]
There is a [suggestion/discussion about this on their Github repo](https://github.com/Gamemode4Dev/GM4_Datapacks/issues/385) about the XP storage having the ability to mend tools

**Reply by aisukuma** [reply]
I've added [Mending Tanks](https://gm4.co/modules/mending-tanks) to the server. You can transfer your XP from the ender chest and stand on top of a liquid tank to deposit 30 levels worth of XP to make it an experience tank. After that, you can put your damaged item on the first slot and it'll repair it every 8 ticks.

**Comment by catter1** [comment]
This is a [Gamemode4 module](https://gm4.co/modules/xp-storage). Technically possible to change, but would have to end the datapack. Not saying we won't, though.
